curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2c2fedbd5f14055b7e7b.md

```diff
@@ -0,0 +1,58 @@
+---
+id: 656d2c2fedbd5f14055b7e7b
+title: Task 163
+challengeType: 19
+dashedName: task-163
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Jake: Well, five o'clock. I guess this is it for your first day. How was it?
+-->
+
+# --description--
+
+`I guess` is a phrase used when someone is not completely sure but wants to make an assumption or express an opinion.
+
+# --question--
+
+## --text--
+
+In the context of the audio, what does `I guess` mean?
+
+## --answers--
+
+`Jake is absolutely certain.`
+
+### --feedback--
+
+`Guess` means there's some uncertainty.
+
+---
+
+`Jake is not sure, but she thinks.`
```

Comment:
Two issues in the answer "Jake is not sure, but she thinks." Change "she" to "he" and maybe rephrase: "Jake is not sure. He thinks what he is saying is true."